sql/core/src/test/resources/sql-tests/inputs/postgreSQL/window_part3.sql

```diff
@@ -400,10 +400,10 @@ SELECT ntile(0) OVER (ORDER BY ten), ten, four FROM tenk1;
 -- filter
 
 -- [SPARK-28500] Adds support for `filter` clause
```

Comment:
Pg supports this query. If jira for this issue doesn't exist in [the Pg feature parity ticket](https://issues.apache.org/jira/browse/SPARK-27764), can you file jira and update this comment?
https://github.com/postgres/postgres/blob/REL_12_STABLE/src/test/regress/expected/window.out#L2989-L3000

Reply:
@maropu I created a new jira https://issues.apache.org/jira/browse/SPARK-30182.

Reply:
Yea, thanks! Then, plz update the comment.

Reply:
OK.